src/Components/Categories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// eslint-disable-next-line
```

Comment:
- [ ] Kindly remove the `Eslint-disable` statement from this file and the following files in this application:

- `src/redux/ConfigureStore.js`
- `src/Components/Book.js`
- `src/index.js`
- `src/App.js`

The _**Eslint**_ helps us write clean, maintainable and scalable codes. Using `Eslint-disable` statements defeats this purpose. 

#### Hint: By the time you correct the paths of the import statements I mentioned in other comments, you won't need the `Eslint-disable` comment for most of them. For the **_Eslint proptypes error_**, you need to establish the type of `props` you are using in the file. Kindly look at this [article](https://reactjs.org/docs/typechecking-with-proptypes.html) for a better understanding. 👍 😎